src/main/java/org/elasticsearch/action/admin/indices/analyze/AnalyzeRequest.java

```diff
@@ -61,11 +61,11 @@ public AnalyzeRequest(String index) {
         this.index(index);
     }
 
-    public String text() {
+    public String[] text() {
```

Comment:
I think this breaks the java api :) we have to mark as breaking then and push only to master (which was the case already I think)